LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -5607,5 +5610,328 @@ LocLongDescription="Free <Ability:WeaponName/> attacks on any enemies that enter
+LocLongDescription="Once per turn, gain an additional move action after taking a standard shot at a flanked or exposed target with your primary weapon."
+LocHelpText="Once per turn, gain an additional move action after taking a standard shot at a flanked or exposed target with your primary weapon."
+LocPromotionPopupText="<Bullet/> Once per turn, gain an additional move action after taking a standard shot at a flanked or exposed target with your primary weapon. Works with hit and run.<br/>
+
+[LW_NeutralizingAgents X2AbilityTemplate]
+LocFriendlyName="Neutralizing Agents"
+LocFlyOverText="Neutralizing Agents"
+LocLongDescription="Aid protocol now neutralizes fire, poison, acid and bleeding."
+LocHelpText="Aid protocol now neutralizes fire, poison, acid and bleeding"
+LocPromotionPopupText="<Bullet/> Aid protocol now neutralizes fire, poison, acid and bleeding."<br/>
+
+[LW_SurvivalInstinct X2AbilityTemplate]
+LocFriendlyName="Survival Instinct"
+LocLongDescription="Gain <Ability:+Defense/> Defense and <Ability:+Crit/> Crit chance while injured."
+LocHelpText="Gain <Ability:+Defense/> Defense and <Ability:+Crit/> Crit chance while injured."
+LocFlyOverText="Survival Instinct"
+LocPromotionPopupText="<Bullet/> The bonuses are lost if you are healed back to full health.<br/>"
+
+[LW_Weaponhandling X2AbilityTemplate]
+LocFriendlyName="Weapon Handling"
+LocLongDescription="Reduces the Aim penalty for short-range <Ability:WeaponName/> shots by <Ability:WPN_HANDLING_MODIFIER/>%."
+LocHelpText="Reduces the Aim penalty for short-range <Ability:WeaponName/> shots by <Ability:WPN_HANDLING_MODIFIER/>%."
+LocFlyOverText="Weapon Handling"
+LocPromotionPopupText="<Bullet/> Point Blank does not affect the aim penalty from Snap Shot.<br/>"
+
+[LW_ThatsCloseEnough X2AbilityTemplate]
+LocFriendlyName="That's Close Enough"
+LocFlyOverText="That's Close Enough"
+LocLongDescription="During enemy turns, fire your arcthrower at any visible enemy within <Ability:THATS_CLOSE_ENOUGH_TILE_RANGE/> tiles who moves or fires. Shares cooldown with arc thrower stun."
+LocHelpText="During enemy turns, fire your arcthrower at any visible enemy within four tiles who moves or fires."
+LocPromotionPopupText="<Bullet/> Will not trigger if soldier is concealed.<br/><Bullet/> This ability can trigger <Ability:THATS_CLOSE_ENOUGH_PER_TARGET_COOLDOWN/> time(s) per turn.<br/>"
+
+[LW_QuickZap X2AbilityTemplate]
+LocFriendlyName="Quick Zap"
+LocFlyOverText="Quick Zap"
+LocLongDescription="Activate this ability to make your next arcthrower shot a free action. <Ability:RAPID_STUN_COOLDOWN/> turn cooldown."
+LocHelpText="Activate this ability to make your next arcthrower shot a free action. <Ability:RAPID_STUN_COOLDOWN/> turn cooldown."
+LocPromotionPopupText="<Bullet/> Activate this ability to make your next arcthrower shot a free action. <Ability:RAPID_STUN_COOLDOWN/> turn cooldown.<br/><Bullet/> Applies to the abilities Stun, Arc Pulser as well as Chain Lightning.<br/>"
```

Comment:
```suggestion
LocLongDescription="Activate this ability to make your next arc thrower shot a free action. <Ability:RAPID_STUN_COOLDOWN/>-turn cooldown."
LocHelpText="Activate this ability to make your next arc thrower shot a free action. <Ability:RAPID_STUN_COOLDOWN/>-turn cooldown."
LocPromotionPopupText="<Bullet/> Activate this ability to make your next arc thrower shot a free action.<Bullet/> Has a <Ability:RAPID_STUN_COOLDOWN/>-turn cooldown.<br/><Bullet/> Applies to the abilities Stun, Arc Pulser and Chain Lightning.<br/>"
```

Does this work for the heal as well? And Arc Pulser isn't really a separate ability from the player's perspective, so I'm not sure it should be mentioned as if it is.

Reply:
@Grobobobo Would appreciate an answer to this one, if you can.